server/utils/stigLibrary.ts

```diff
@@ -0,0 +1,291 @@
+import * as fs from "fs";
+import * as path from "path";
+import AdmZip from "adm-zip";
```

Comment:
might be worthwhile investigating if we want to swap to 'archiver'

questions: does it only work in a node context or can it work from a browser context too?  can it reduce the number of dependencies that you have (since it has built in support for both zip and tar)?